example/demo_survey/src/server/serverFieldUpdate.ts

```diff
@@ -28,7 +31,33 @@ export default [
             } else if (value === '1133-1133') {
                 // Special access code to test url redirection
                 return [{}, 'https://github.com/chairemobilite/evolution/']
+            } else if (value.startsWith('222')) {
+                // Use the update operation to set the city to Calgary, but 5 seconds later
+                registerUpdateOperation({
+                    opName: 'city',
+                    opUniqueId: 1,
+                    operation: () => new Promise((resolve) => {
+                        setTimeout(() => {
+                            resolve({
+                                'home.city': 'Calgary'
+                            });
+                        }, 5000);
+                    })
+                });
+                // Use the update operation to set the province to Alberta, but 10 seconds later
+                registerUpdateOperation({
+                    opName: 'province',
+                    opUniqueId: 1,
+                    operation: () => new Promise((resolve) => {
+                        setTimeout(() => {
+                            resolve({
+                                'home.region': 'Alberta'
```

Comment:
Normally, the value starts with a lowercase letter (alberta, and not Alberta). Same for Calgary.

Reply:
In the demo_survey, these are text values entered by the user, so city and province start with uppercase. That would be right if this was a choice like in mj.